### PR TITLE
CLDR-15924 Fix ta and si person names

### PR DESCRIPTION
**Pull request by macchiati**
CLDR-15924

- [ ] This PR completes the ticket.

### CODE REVIEW

**Review by pedberg-icu — DISMISSED** [review]

**Comment by macchiati** [comment]
Just had to resolve merge conflicts. Should be ok once the tests finish.

File: common/main/si.xml [hunk 1]
```diff
@@ -9654,7 +9654,7 @@ Warnings: All cp values have U+FE0F characters removed. See /annotationsDerived/
 			<namePattern>{surname} {given-informal}</namePattern>
```
Comment (mimckenna) [inline common/main/si.xml:9654]: this is for a few lines up.
In Sinhala, we should add _prefix_ in long, referring, formal
```
		<personName order="surnameFirst" length="long" usage="referring" formality="formal">
			<namePattern>{prefix} {surname} {given} {given2} {suffix}</namePattern>
		</personName>
```
see: https://cmb.ac.lk/honorary-degrees/ - si_Latn - formal presentation
https://www.parliament.lk/uploads/documents/paperspresented/annual-report-university-of-jaffna-2017-si.pdf - list of faculty

File: common/main/si.xml [hunk 2]
```diff
@@ -9588,19 +9588,19 @@ Warnings: All cp values have U+FE0F characters removed. See /annotationsDerived/
 		<featureName type="zero">කපහරු බිංදු</featureName>
 	</typographicNames>
 	<personNames>
-		<nameOrderLocales order="givenFirst">und si</nameOrderLocales>
-		<nameOrderLocales order="surnameFirst">ko te vi yue zh</nameOrderLocales>
+		<nameOrderLocales order="givenFirst">und</nameOrderLocales>
+		<nameOrderLocales order="surnameFirst">ko si te vi yue zh</nameOrderLocales>
```
Comment (mimckenna) [inline common/main/si.xml:9592]: Add 'ta' to _surnameFirst_ list
```
<nameOrderLocales order="surnameFirst">ko si ta te vi yue zh</nameOrderLocales>
```

**Review by mimckenna — CHANGES_REQUESTED** [review]
A few requested changes 
- be sure both 'si' and 'ta' added to surnameFirst list.
- add `{prefix}` to long, referring, formal

**Comment by macchiati** [comment]
I'd like to get this merged, then follow up with another PR with fixes for Mike's comments. @pedberg-icu can you stamp it?

**Review by pedberg-icu — APPROVED** [review]

**Comment by mimckenna** [comment]
Looks OK for release - we can patch later if Tamil and Sri Lankan folks want `{prefix}` for referring, formal.

**Pull request merged**